pkg/cache/clusterqueue.go

```diff
@@ -237,42 +247,53 @@ func (c *clusterQueue) inactiveReason() (string, string) {
 		return "Terminating", "Can't admit new workloads; clusterQueue is terminating"
 	case pending:
 		reasons := make([]string, 0, 3)
+		messages := make([]string, 0, 3)
 		if c.isStopped {
 			reasons = append(reasons, "Stopped")
+			messages = append(messages, "Is stopped.")
 		}
-		if c.hasMissingFlavors {
+		if len(c.missingFlavors) > 0 {
 			reasons = append(reasons, "FlavorNotFound")
+			messages = append(messages, fmt.Sprintf("References missing ResourceFlavor %v.", c.missingFlavors))
 		}
-		if c.hasMissingOrInactiveAdmissionChecks {
-			reasons = append(reasons, "CheckNotFoundOrInactive")
+		if len(c.missingAdmissionChecks) > 0 {
+			reasons = append(reasons, "AdmissionCheckNotFound")
```

Comment:
+1 for splitting `CheckNotFoundOrInactive` into dedicated conditions. Let's just update the release note to add:

```
ACTION REQUIRED: If you were watching for the reason `CheckNotFoundOrInactive` in the ClusterQueue condition, watch `AdmissionCheckNotFound` and `AdmissionCheckInactive` instead.
```

Reply:
done